### PR TITLE
Add a 'name' attribute to acl_target and acl_group elements

### PR DESCRIPTION
**Pull request by waltdisgrace**
@kgaillot I followed Steps 1-8 of https://github.com/ClusterLabs/pacemaker/blob/master/xml/README.md to make this PR. Let me know if it looks right...

### CODE REVIEW

File: xml/acls-2.1.rng [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kgaillot) [inline xml/acls-2.1.rng:1]: The README is somewhat wrong on one point -- the file name needs to increment the highest number of any schema file, not just the file being edited, so this file should be acls-3.8.rng. Please correct the README too ...

File: xml/acls-2.1.rng [hunk 2]
```diff
@@ -22,6 +23,7 @@
           <element name="acl_group">
             <!-- Here 'id' is the name of a unix group -->
             <attribute name="id"><data type="ID"/></attribute>
+            <attribute name="name"><text/></attribute>
```
Comment (kgaillot) [inline xml/acls-2.1.rng:26]: The new attributes should be optional. The tests should succeed after fixing that

**Review by kgaillot — COMMENTED** [review]
Good start

File: xml/acls-2.1.rng (hunk 2 above)
Comment (waltdisgrace) [reply xml/acls-2.1.rng]: Hmm, still 70 failures.

**Review by kgaillot — COMMENTED** [review]
Everything but the regression test commit looks good. I'm guessing you didn't run "make" before updating the regression tests, so it had some files left over from previous attempts. Before the regression test commit, try "make -C xml clean && make", then try updating the regression tests again. make in the xml directory dynamically creates the pacemaker-*.rng files based on all the others.

**Review by kgaillot — APPROVED** [review]
Perfect, thanks :)

**Pull request merged**